libr/core/disasm.c

```diff
@@ -1179,6 +1179,17 @@ static void ds_begin_line(RDisasmState *ds) {
 	if (ds->pj) {
 		pj_o (ds->pj);
 		pj_kn (ds->pj, "offset", ds->vat);
+		if (ds->core->anal->reflines) {
+			RAnalRefline *ref;
+			RListIter *iter;
+			// XXX Probably expensive
```

Comment:
thats not probably, i'm pretty sure its an expensive operation. use this:

                list = r_anal_xrefs_get (core->anal, addr);